cmd/command_test.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/spf13/cobra"
+)
+
+func executeCommand(root *cobra.Command, args ...string) (output string, err error) {
+	_, output, err = executeCommandC(root, args...)
```

Comment:
What is the purpose of this wrapper function? Why not call `executeCommandC()` directly from the test?

Reply:
@imiric The `executeCommandC` will be used when you test sub command, even we didn't have a test for any sub commands, but it's better to add it here.

Note that I actually copied it from original repo https://github.com/spf13/cobra/blob/b80588d523ec50c7fee20218426cf2ff70920f06/command_test.go#L16

:smile: